### DISCUSSION TITLE
Dec 2023 Review - PR6: Properties and Graphs of Rational Functions

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#48] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/PR6.html

Author: @AbbyANoble 

Primary Reviewer: @tdegeorge  (feedback by Dec 11 morning)

**Comment by siwelwerd** [comment]
Abby's additions from this morning are now integrated into main and viewable at the link above

**Comment by tdegeorge** [comment]
Activity 1.6.2 (part c): I would change the question so it says "Which one(s)?" because just having "Which one?" kinda gives it away that there is one answer. It might also provide some discussion as to why r(1) is not a zero.

**Comment by tdegeorge** [comment]
I really like activity 1.6.4. The only thing I saw is that there is a typo in part f (should be "than" rather than "that").

**Comment by tdegeorge** [comment]
It would probably be a good idea to have a horizontal asymptote activity like the one you created for the vertical asymptotes.

**Reply by AbbyANoble** [reply]
Good call! HA's clearly slipped my mind! Will work on that.

**Comment by tdegeorge** [comment]
Activity 1.6.10: Would it be possible to have one or two of the graphs have holes in them?

**Reply by AbbyANoble** [reply]
Yep! Or possibly another match the graph activity where there are holes involved?

**Comment by tdegeorge** [comment]
The last activity for the section should probably have a "sketch a graph with these properties" to align with the objective.

**Comment by AbbyANoble** [comment]
I've made some of these changes, but had a few things pop up this week I wasn't expecting so it is not complete. Will get it done before we meet again! 

@siwelwerd @StevenClontz  - What are your off days for the break? I'll avoid PRs and questions during that time and wait until you are out of holiday mode!

**Reply by siwelwerd** [reply]
My kid is still in school next week, so feel free to ping me then.  But I'll be out of the office after that and then at JMM until kiddo's school starts back up 1/8.

**Comment by siwelwerd** [comment]
I went ahead and merged in what was there.  A few questions/comments I had:

- I wonder if [Activity 1.6.3](https://teambasedinquirylearning.github.io/precalculus/2024/PR6.html#activity-rf-domain-and-zeros) could be streamlined -- instead of asking parts (b), (c) and (d) separately, maybe ask them to classify those four points as providing information about the domain, zeroes, or nonzero points.
- Is [Definition 1.6.4](https://teambasedinquirylearning.github.io/precalculus/2024/PR6.html#def-rf-domain) really a definition?  I ask because (1) Domain is used in the previous activity, and (2) My inclination is to use a more general definition of domain, and have what is stated here as an observation that is roughly established by the previous activity.

**Reply by AbbyANoble** [reply]
I agree about 1.6.4. I do think that is more of an observation about a particular type of function's domain. I'll update that.

I'm not sure how to rephrase 1.6.3 to streamline. My thoughts in separating them were to lead to specific answers, and then talk about each of them one at a time. What were you thinking?

**Reply by siwelwerd** [reply]
I was thinking something along the lines of

(a) Find r(1), r(2), r(3), r(4)
(b) Label each of these four points as giving us information about the DOMAIN of r(x), information about the ZEROES of r(x), or NEITHER.
